src/main/scala/com/databricks/spark/sql/perf/mllib/MLBenchContext.scala

```diff
@@ -0,0 +1,39 @@
+package com.databricks.spark.sql.perf.mllib
+
+import java.util.Random
+
+import com.databricks.spark.sql.perf.{MLParams}
+import org.apache.spark.sql.SQLContext
+
+
+/**
+ * All the information required to run a test.
+ *
+ * @param extraParams
+ * @param sqlContext
+ */
+case class MLBenchContext(
```

Comment:
That class is still useful I find for passing the sql context and the spark context around, I find.